src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -2781,8 +2781,17 @@ internal static int LengthInBufferCells(string str, int offset, bool checkEscape
                     escapeSequenceAdjustment += ControlSequenceLength(str, ref i);
                 }
             }
+#if !UNIX
```

Comment:
Isn't this a problem on Linux/Mac as well?

Reply:
I suppose but I cannot test.
Is there any reason why the `LengthInBufferCells` function is disabled on unix?

Reply:
I don't know for sure, but I would guess the original code depended on Windows apis and when porting to non-Windows, they punted on porting that code until a later time.

Reply:
Exlcuding `LengthInBufferCells` from that block since it seems to be harmless.